src/nix/verify.cc

```diff
@@ -41,7 +41,7 @@ struct CmdVerify : StorePathsCommand
         addFlag({
             .longName = "sigs-needed",
             .shortName = 'n',
-            .description = "Require that each path has at least *n* valid signatures.",
+            .description = "Require that each path has is signed by *n* different keys.",
```

Comment:
I guess it should still say "at least" here? Also the "has" from the previous sentence is still there.

```suggestion
            .description = "Require that each path is signed by at least *n* different keys.",
```